### PR TITLE
Improve API docs

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2288

## Examples
N/A

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/client.bal [hunk 1]
```diff
@@ -131,13 +123,14 @@ public type SecureSocket record {|
     crypto:TrustStore cert?;
 |};
 
-# Oracle database connection parameters.
+# Provides a set of configuration related to Oracle database connection.
```
Comment (kaneeldias) [inline ballerina/client.bal:126]:
```suggestion
# Provides a set of configurations related to Oracle database connection.
```

File: ballerina/types.bal [hunk 2]
```diff
@@ -77,7 +77,7 @@ public type NestedTableType record {|
     ArrayValueType? elements;
 |};
 
-# Represents OBJECT TYPE Oracle DB field.
+# Represents OBJECT TYPE  parameter in `sql:ParameterizedQuery`.
```
Comment (daneshk) [inline ballerina/types.bal:80]:
```suggestion
# Represents OBJECT TYPE parameter in `sql:ParameterizedQuery`.
```

File: ballerina/types.bal [hunk 3]
```diff
@@ -89,9 +89,9 @@ public distinct class ObjectTypeValue {
     }
 }
 
-# Represents VARRAY Oracle DB field.
+# Represents VARRAY type  parameter in `sql:ParameterizedQuery`.
```
Comment (daneshk) [inline ballerina/types.bal:92]:
```suggestion
# Represents VARRAY type parameter in `sql:ParameterizedQuery`.
```

File: ballerina/types.bal [hunk 4]
```diff
@@ -101,9 +101,9 @@ public distinct class VarrayValue {
     }
 }
 
-# Represents Nested Table Oracle DB field.
+# Represents Nested Table type  parameter in `sql:ParameterizedQuery`.
```
Comment (daneshk) [inline ballerina/types.bal:104]:
```suggestion
# Represents Nested Table type parameter in `sql:ParameterizedQuery`.
```

File: ballerina/types.bal [hunk 5]
```diff
@@ -17,10 +17,10 @@
 import ballerina/jballerina.java;
 import ballerina/sql;
 
-# Represents the type that can be either minus one or plus one.
+# Represents the Oracle sign.
```
Comment (daneshk) [inline ballerina/types.bal:20]:
```suggestion
# Represents the Oracle (+/-) sign.
```

File: ballerina/types.bal (hunk 4 above)
Comment (niveathika) [reply ballerina/types.bal]: Fixed in 1d2a86b

File: ballerina/types.bal (hunk 3 above)
Comment (niveathika) [reply ballerina/types.bal]: Fixed in 1d2a86b

File: ballerina/types.bal (hunk 2 above)
Comment (niveathika) [reply ballerina/types.bal]: Fixed in 1d2a86b

**Review by daneshk — APPROVED** [review]

**Pull request merged**